src/main/java/com/squareup/osstrich/JavadocPublisher.java

```diff
@@ -161,17 +158,26 @@ private String findRelativePath(File directory, String artifactId) {
     if (!directory.isDirectory()) {
       throw new IllegalArgumentException(directory + " is not a directory!");
     }
+
+    // Look for Javadoc in the current directory.
     if (new File(directory, "index.html").exists()) {
       return directory.getName();
-    } else {
-      // Look for subdirectories of the same name as the artifact and go a level deeper
-      File possibleSubDir = new File(directory, artifactId);
-      if (possibleSubDir.isDirectory()) {
-        return directory.getName() + File.separator + findRelativePath(possibleSubDir, artifactId);
-      }
-      throw new RuntimeException("Could not find a valid indexed path for " + artifactId + ". Files are "
-              + Arrays.toString(Objects.requireNonNull(directory.listFiles())));
     }
+
+    // Look for subdirectories of the same name as the artifact and go a level deeper
+    File artifactDir = new File(directory, artifactId);
+    if (artifactDir.isDirectory()) {
+      return directory.getName() + File.separator + findRelativePath(artifactDir, artifactId);
+    }
+
+    // For Kotlin multiplatform look for a subdirectory named 'jvm' and go a level deeper.
```

Comment:
There also could be both jvm and android-specific platform modules.

Reply:
Oooh tricky. Recommendations?